Core/CKAN.schema

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
Is it impossible to move the old CKAN.schema file to this new location? It will be inconvenient to have to maintain two files identically going forward.

Reply:
It could be a symbolic link instead of a separate file, surely?